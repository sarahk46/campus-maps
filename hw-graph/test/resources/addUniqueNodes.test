# Tests that a graph can be created and that five unique nodes can be added to it.

# Create a graph
CreateGraph graph1

# Add five unique nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3
AddNode graph1 n4
AddNode graph1 n5