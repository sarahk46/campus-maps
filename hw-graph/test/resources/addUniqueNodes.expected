# Tests that a graph can be created and that five unique nodes can be added to it.

# Create a graph
created graph graph1

# Add five unique nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1
added node n4 to graph1
added node n5 to graph1